spec.bs

```diff
@@ -588,6 +589,14 @@ members:
         credential(s). The caller does not need to specify the
         [[#sctn-payment-extension-registration| payment extension]]; it is added
         automatically.
+
+    :  <dfn>localeHint</dfn> member
+    :: An optional list of **TODO, ISO-something?** locale descriptors, in
```

Comment:
I would say maybe something like:

> An optional list of well-formed [BCP47] language tags, in decreasing order of priority, that identify the `locale` preference(s) of the website, i.e. a `language priority list` [RFC4647] which the relying party can use to perform `language negotiation` and locale-affected formatting with the caller.

(note that 4647 is part of BCP47)
(you can just link the terms `locale`, `language negotiation` and `language priority list` to the I18N-GLOSSARY: if you import our glossary you can just <a> link them)

Reply:
Done, though I'm not familiar with what it means to import the spec in bikeshed. I thought exported definitions should 'just work' with auto-linking (`[= ... =]`), but I wasn't able to reference e.g., `locale` and had to manually put them as [custom definitions](https://tabatkins.github.io/bikeshed/#custom-dfns).

Happy to fix that if you know how :)